bootstrap/boot.ps1

```diff
@@ -1007,7 +1007,37 @@ function Install-PlatformModules
 #########################################################################################################
 #region Helper functions
 #########################################################################################################
+<#
+ Housekeeping function for bootstrap
+#>
+function Start-BootstrapHosekeeping
```

Comment:
This looks like a typo...I'm sure that was supposed to be Start-BootstrapHousekeeping?